packages/@aws-cdk/toolkit-lib/test/actions/deploy.test.ts

```diff
@@ -53,10 +54,9 @@ describe('deploy', () => {
 
   test('request response when require approval is set', async () => {
     // WHEN
+    ioHost.requireDeployApproval = RequireApproval.ANY_CHANGE;
```

Comment:
I don't understand that effect this has on the test?

Reply:
this is how we set require approval now. i can add a comment in the code